code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2254,7 +2254,7 @@
 		return 1
 
 	if(prob(33))
-		M.take_organ_damage(REM, 0)
+		M.take_organ_damage(REM, 0, ignore_inorganics = TRUE)
```

Comment:
The type of `M` here is `/mob/living`. However, you only defined `var/ignore_inorganics` for `/mob/living/carbon/human`, so this will runtime for anything not humans.

Reply:
Added the variable to mob-level take_organ_damage